spec/helpers/application_helper_spec.rb

```diff
@@ -1641,4 +1641,23 @@
       ).to eq('host_services')
     end
   end
+
+  describe "#multiple_relationship_link" do
+    context "When record is a Container Provider" do
```

Comment:
shouldn't there be also a test that checks that when user arrives to provider summary page there's a 200 OK?
something like this:
https://github.com/ManageIQ/manageiq/blob/master/spec/controllers/ems_container_controller_spec.rb#L14
I am wondering btw how is the test for "show" in containers provider is still working given the changes in #7950